root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Resources/PublicExceptionMessageResource.resx

```diff
@@ -180,6 +180,9 @@
   <data name="NOT_IMPLEMENTED" xml:space="preserve">
     <value>This processing is not Implemented.</value>
```

Comment:
@daisukenishino 

We have changed "Implemented" -> "implemented" in  [PublicExceptionMessageResource.resx file](https://github.com/SymphonyTeleca/OpenTouryoTemplates/pull/162/commits/ce92daac6038eee21bdf31910500b6014ff3d92e#diff-04a2897f285535a9c7049234ba03c52bR181)

Please review [#162](https://github.com/SymphonyTeleca/OpenTouryoTemplates/pull/162) and merge.